### PR TITLE
(feat) Set Urgency and Priority for an item.

### PR DESCRIPTION
**Pull request by mdhaman**
SD-2627, SD-3159 and SD-3160
- Urgency values will be 1 to 5. 
- NewsValue is no longer needed.
- Priority will be as described in ticket SD-2627

### CODE REVIEW

File: superdesk/io/newsml_1_2.py [hunk 1]
```diff
@@ -38,6 +38,9 @@ def parse_message(self, tree, provider):
             if parsed_el is not None:
                 item['ingest_provider_sequence'] = parsed_el.text
 
+            parsed_el = tree.find('NewsEnvelope/Priority')
+            item['Priority'] = self.map_priority(parsed_el.text if parsed_el else None)
```
Comment (sivakuna-aap) [inline superdesk/io/newsml_1_2.py]: this should be `item['priority']` not `item['Priority']`. 'p' is capital, should be small.

**Comment by amagdas** [comment]
:+1: except Siva's comment

**Comment by mdhaman** [comment]
@amagdas @petrjasek this PR changes the Priority from `str` to `int`. 
Do you think we need to drop the database?

**Comment by amagdas** [comment]
Might be a good idea.

File: superdesk/io/__init__.py [hunk 2]
```diff
@@ -112,6 +112,18 @@ def set_dateline(self, item, city=None, text=None):
...
             item['dateline']['text'] = text
 
+    def map_priority(self, source_priority):
+        """
+        Maps the source priority to superdesk priority
+        :param str source_priority:
+        :return int: priority of the item
+        """
+        if source_priority and source_priority.isdigit():
+            if int(source_priority) in [1, 2, 3]:
```
Comment (amagdas) [inline superdesk/io/__init__.py]: What's up with 4? :)

Comment (sivakuna-aap) [inline superdesk/io/__init__.py]: Well the requirement for now is upto 3 only. :smirk:.

Comment (amagdas) [inline superdesk/io/__init__.py]: Then what's up with 5 :smile:  ?

Comment (sivakuna-aap) [inline superdesk/io/__init__.py]: anything which doesn't fall between 1-3 is 5 :smirk:

Comment (mdhaman) [inline superdesk/io/__init__.py]: Verified with Lasky, everything in the cyber system goes out as 5. 1-3 priority level are introduced in Superdesk and rest is 5. :smirk:

**Pull request merged**